addons/mail/static/src/components/composer/composer.xml

```diff
@@ -88,6 +88,7 @@
                     />
                     <div class="o_Composer_buttons" t-att-class="{ 'o-composer-is-compact': props.isCompact, 'o-mobile': env.messaging.device.isMobile }">
                         <div class="o_Composer_toolButtons"
+                            t-if="!composer.isDisabled"
```

Comment:
Here we would need to clarify the spec with PO, but I don't think we want to remove everything. I think it would be more clear if we just disabled them instead.